projects/ccf-rui/src/app/core/store/model/model.state.ts

```diff
@@ -256,6 +256,9 @@ export class ModelState extends NgxsImmutableDataRepository<ModelStateModel> {
   @DataAction()
   setOrgan(organ: OrganInfo): void {
     this.ctx.patchState({ organ });
+    if (organ.side) {
```

Comment:
Don't call dataactions from other data actions. Instead patch the state with the side unless you can remove side from the model completely (which I think we can)